ui/js/models/perf/alerts.js

```diff
@@ -199,7 +199,7 @@ treeherder.factory('PhAlerts', [
             // the summary should only incorporate those. if there
             // aren't, then use all of them (that aren't downstream,
             // see above)
-            if (_.some(_.map(alertsInSummary, 'is_regression'))) {
```

Comment:
I agree! However eslint won't allow the `!!` syntax.
So we could have something like:
`if (alertsInSummary.filter(alert => alert.is_regression).length > 0) {`

Is that ok?

Reply:
Also, to avoid a double call of `filter(),` I propose this:
```
const regressions = alertsInSummary.filter(alert => alert.is_regression);
    if (regressions.length > 0) {
        alertsInSummary = regressions;
}```

Reply:
Sure, that sounds good to me!  :)  I'm always up for saving some repeated code.